soccer/SystemState.cpp

```diff
@@ -139,7 +139,7 @@ RJ::Time Ball::estimateTimeTo(const Geometry2d::Point& point,
 
 double Ball::estimateSecondsTo(const Geometry2d::Point &point) const {
     const auto time = estimateTimeTo(point);
-    return RJ::Seconds(RJ::now() - time).count();
+    return RJ::Seconds(time - RJ::now()).count();
```

Comment:
I think this was a bug. `estimateTimeTo` returns the current time + the time for the ball to move that distance. To get the time between that estimate and now, you have to subtract now otherwise the time is always negative